src/main/java/com/Model/User.java

```diff
@@ -1,57 +1,78 @@
 package com.Model;
 
-import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
 import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
 
 @Entity
-@Inheritance (strategy = InheritanceType.JOINED)
-public class User {
+public class User implements UserDetails {
 
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private long id;
+
     private String emailAddress;
     private String password; //Encrypted password
-    private boolean isEnabled; //False if user has not confirmed email address yet
-    private boolean isVerified;
 
-    @Autowired
     public User(){
         this("unknown", "unknown");
     }
 
     public User (String emailAddress, String password){
         this.emailAddress = emailAddress;
         this.password = password;
```

Comment:
this is where you could do the encryption and just store the encrypted version.

Reply:
Good idea! I will do this way.